Backend/database/liveContext.go

```diff
@@ -0,0 +1,54 @@
+package database
```

Comment:
Might be a good idea to include some header comments in these files, because i think these ones are especially likely to prompt next  year's people to ask for help